<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConnectionWindow</class>
 <widget class="QWidget" name="ConnectionWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>352</width>
    <height>284</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Connection Settings</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Connection Type</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QRadioButton" name="rbGVRET">
        <property name="text">
         <string>GVRET Serial</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="rbKvaser">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Kvaser</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="rbSocketCAN">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>SocketCAN</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Port:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QComboBox" name="cbPort"/>
   </item>
   <item row="3" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>CAN0 Speed</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>CAN1 Speed</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QComboBox" name="cbSpeed0"/>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="cbSpeed1"/>
     </item>
    </layout>
   </item>
   <item row="4" column="0">
    <widget class="QCheckBox" name="ckSingleWire">
     <property name="text">
      <string>Set CAN1 to Single Wire Mode</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QCheckBox" name="ckCompleteCode">
     <property name="text">
      <string>Append CompleteCode for No Device Exist</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QPushButton" name="btnOK">
     <property name="text">
      <string>Update Settings</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
